Index: CA302_Project1.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"JAVA_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\" inherit-compiler-output=\"false\">\r\n    <output url=\"file://$MODULE_DIR$/out/production/CA302_Project1\" />\r\n    <output-test url=\"file://$MODULE_DIR$/out/test/CA302_Project1\" />\r\n    <exclude-output />\r\n    <content url=\"file://$MODULE_DIR$\">\r\n      <sourceFolder url=\"file://$MODULE_DIR$/src\" isTestSource=\"false\" />\r\n    </content>\r\n    <orderEntry type=\"jdk\" jdkName=\"corretto-21\" jdkType=\"JavaSDK\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n    <orderEntry type=\"library\" name=\"lib\" level=\"project\" />\r\n    <orderEntry type=\"library\" name=\"gson\" level=\"project\" />\r\n    <orderEntry type=\"library\" name=\"xerial.sqlite.jdbc\" level=\"project\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CA302_Project1.iml b/CA302_Project1.iml
--- a/CA302_Project1.iml	(revision 13b2fa1b187681ec786693d1876a86c8c71f37a3)
+++ b/CA302_Project1.iml	(date 1776929364227)
@@ -1,16 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="false">
-    <output url="file://$MODULE_DIR$/out/production/CA302_Project1" />
-    <output-test url="file://$MODULE_DIR$/out/test/CA302_Project1" />
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
     </content>
-    <orderEntry type="jdk" jdkName="corretto-21" jdkType="JavaSDK" />
+    <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" name="lib" level="project" />
-    <orderEntry type="library" name="gson" level="project" />
-    <orderEntry type="library" name="xerial.sqlite.jdbc" level="project" />
   </component>
 </module>
\ No newline at end of file
